definitions/features/tar.rb

```diff
@@ -51,13 +51,11 @@ def tar_command(options)
     end
 
     if volume_size
-      split_tar_script = default_split_tar_script
       tar_command << "--tape-length=#{volume_size}"
-      tar_command << "--new-volume-script=#{split_tar_script}"
     end
 
     tar_command << '--overwrite' if options[:overwrite]
-    tar_command << '--gzip' if options[:gzip]
+    tar_command << (options[:gzip] ? '--gzip' : "--new-volume-script=#{default_split_tar_script}")
```

Comment:
Previously this was gated by `volume_size` being true, is that no longer necessary?

Reply:
Yes and no: https://github.com/theforeman/foreman_maintain/pull/680#issuecomment-1405684413 . Technically not necessary, `tar` does cope well (I successfully tested it, anyway an independent verification is welcomed). "Logically" or methodologically it is required (why to treat archive as "split to tapes" every time, even though it is usually not the case?). But I havent found out a simple test to distinguish that ("was the tar archive created with -t option, or not?").